src/DICOM.jl

```diff
@@ -158,21 +159,23 @@ function pixeldata_parse(st, sz, vr, dcm)
         dtype = UInt16
     end
     if !is(dcm,false)
+	# (0028,0010) defines number of rows
         f = lookup(dcm, (0x0028,0x0010))
         if !is(f,false)
-            xr = f.data[1]
+            xr = f.data[1][1]
```

Comment:
I see. The parsing code can probably be re-worked a bit to avoid that, but I'll merge for now and move to JuliaIO.